frontend/screens/BrandRegistrationForm.js

```diff
@@ -10,38 +10,50 @@ import {
   Alert,
 } from "react-native";
 import { Color, Padding, FontSize, Border, FontFamily } from "../GlobalStyles";
-import {API_ENDPOINT} from '@env'
-
+import { API_ENDPOINT } from "@env";
+import MultipleSelectList from "../shared/MultiSelect";
 
 const BrandRegistrationForm = ({ navigation }) => {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
-  const [brandType, setBrandType] = useState("");
   const [username, setUsername] = useState("");
+  const [selected, setSelected] = useState([]);
+
+  const data = [
+    { key: "grocery", value: "Grocery" },
+    { key: "electronics", value: "Electronics" },
+    { key: "fashion", value: "Fashion" },
+    { key: "toys", value: "Toys" },
+    { key: "beauty", value: "Beauty" },
+    { key: "home-decoration", value: "Home Decoration" },
+    { key: "fitness", value: "Fitness" },
+    { key: "education", value: "Education" },
+    { key: "others", value: "Others" },
+  ];
 
   const handleSubmit = async () => {
     // Construct the payload
     const payload = {
       email,
       password,
-      category: [brandType],
+      category: selected,
       name: username,
     };
     const emailRegex = /\S+@\S+\.\S+/;
-    if(!emailRegex.test(email)){
+    if (!emailRegex.test(email)) {
       Alert.alert("Error", "Please enter a valid email address");
       return;
     }
-    if (!email || !password || !brandType || !username) {
+    if (!email || !password || !selected.length || !username) {
       Alert.alert("Error", "Please fill all the fields");
       return;
     }
-    if(password.length<8){
-      Alert.alert("Error", "Password should be atleast 8 characters long");
+    if (password.length < 8) {
+      Alert.alert("Error", "Password should be at least 8 characters long");
       return;
     }
     try {
```

Comment:
Can we please move the api calls under `src/controller/signupController.js` file? It will be organised from the start that way